README.md

```diff
@@ -2,197 +2,230 @@
 
-When logging in, you will be asked for **3** things:
+# Wanna know something cool? You can customize your prompt
 
-- The `account` name of the store you wish to work on
-- Your VTEX ID `e-mail`
-- Your VTEX ID `password`
+You can configure your **terminal prompt** to display relevant information about your current context, like which account you're logged into and which `workspace` you are currently using.
 
-If you wish to work on another `account`, logout and login again with that `account`.
+Just like knowing which `git` branch you're currently in, having this info in your prompt you help you avoid mistakes and be faster when using VTEX IO.
 
-Note that `link` and `publish` implicitly checks if you're logged, and if you're not, it asks your credentials before proceeding.
+## How to do so? 
 
+#### `bash` users
 
-## How do I develop an app locally?
+If you are a bash user you can start customizing your command prompt by running the following command:
 
-To develop an app locally, open the directory where your VTEX app is and then type:
+```sh
+echo "source $(npm root -g)/vtex/plugins/bash/vtex.bash" >> ~/.bashrc
+echo "source $(npm root -g)/vtex/plugins/bash/prompt.bash" >> ~/.bashrc
+```
+
+> 😉 For fine grained control, use only the first script and add `__vtex_ps1` manually to your prompt.
+
+#### `fish` users
+
+If you want a pretty, ready-made Git + VTEX prompt you can copy `plugins/fish/fish_prompt.fish` to the  `~/.config/fish/functions` folder. To see the changes, simply restart your shell by typing `fish`.
 
 ```sh
-vtex link
+cp (npm root -g)/vtex/plugins/fish/fish_prompt.fish ~/.config/fish/functions/
+fish
 ```
 
-Toolbelt will upload all your app files to the developer environment, print an URL for your use and watch for any changes you make to the files, which will be automatically synchronized.
+> 💡 If you installed Toolbelt using `yarn`, you should replace `(npm root -g)` with `(yarn global dir)`.
 
-## How do I ignore specific files or directories in my app so that they are not uploaded?
+---
+
+# Frequently Asked Questions
+
+<details>
+<summary>How do I ignore specific files or directories in my app so that they are not uploaded?</summary>
 
 Create a `.vtexignore` file containing, on each line, paths you wish to ignore.
 If no `.vtexignore` is found, your [.gitignore](http://git-scm.com/docs/gitignore) file is used instead.
 
+</details>
 
-## I don't like the default toolbelt prompt customization. It's ugly or it messes with some of my other configurations, but I still want to be able to see the account and workspace I'm logged into. What do I do?
+<details>
+<summary>Is there another way to customize the prompt?</summary>
 
 You can do so by adding the `vtex_get_account` and `vtex_get_workspace` commands to your PS1 environment variable the way it pleases you.
 
+</details>
+
+
 ---
 ## Contributing
```

Comment:
Remember to change the path of this contributing file

Reply:
@tiagonapoli could you please change the path for me? I tried but don't know why it didn't work 😅